wikipedia-dark.user.css

```diff
@@ -6546,6 +6547,49 @@ regexp("^https://(foundation|donate)\.wikimedia\.org.*$") {
   .list-switcher-element a[role="button"] {
     color: var(--base-color) !important;
   }
+  tr > *[style*="background:#c0cfe4"],
+  tr > *[style*="background:#9999DF"] {
+    background-color: var(--blue-4) !important;
+  }
+  tr > *[style*="background:#a0ade3"],
+  tr > *[style*="background:#ccccff"] {
+    background-color: var(--blue-3) !important;
+  }
+  tr > *[style*="background:#e2e4c0"],
+  tr > *[style*="background:#d0cfa4"] {
+    background-color: var(--yellow-1) !important;
+  }
+  tr > *[style*="background:#c0d8e4"] {
```

Comment:
`#c0d8e4` is on blue spectrum, so maybe a `--blue-3` intead of adding a new yellow var ;)

Reply:
That's right